src/Xamarin.Android.Build.Tasks/Properties/Resources.resx

```diff
@@ -117,6 +117,19 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="APT0002" xml:space="preserve">
+    <value>Invalid file name: It must contain only {0}.</value>
+    <comment>{0} - The regular expression that the file name must match</comment>
+  </data>
+  <data name="APT0003" xml:space="preserve">
+    <value>Invalid file name: It must contain only {0}.</value>
+    <comment>{0} - The regular expression that the file name must match</comment>
+  </data>
+  <data name="XA0100" xml:space="preserve">
+    <value>EmbeddedNativeLibrary '{0}' is invalid in Android Application projects. Please use AndroidNativeLibrary instead.</value>
```

Comment:
To help with what to look at in this PR, I'll mention that the only wording change this time around is to change "project" to "projects" in this XA0100 message.